discordkit/utils/event.ts

```
@@ -0,0 +1,46 @@
+    static build() {
+      return {
+        name: String(config.name),
+        once: config.once || false,
+        execute: async (
+          harmonix: Harmonix,
+          ...args: unknown[]
+        ): Promise<void> => {
+          return Effect.runPromise(
+            Effect.tryPromise(async () => {
+              logInfo(`Executing event ${String(config.name)}`, "event");
+              await this.execute(harmonix, ...(args as any));
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.

(lint/complexity/noThisInStatic)